msal/src/Microsoft.Identity.Client/UIBehavior.cs

```diff
@@ -61,6 +61,12 @@ struct UIBehavior
         /// </summary>
         public static readonly UIBehavior Consent = new UIBehavior("consent");
 
+        /// <summary>
+        /// For B2C scenarios where each policy is a different authority, to prevent UI from being shown
+        /// to a user who has previously acquired a token for a different policy. It is achieved
+        /// by sending <c>prompt=none</c> to the Azure AD service.
+        /// </summary>
+        public static readonly UIBehavior NoPrompt = new UIBehavior("none");
```

Comment:
What are the implications of using this prompt in non-B2C scenarios? Is it something that we want to restrict users from doing, i.e. IF Authority != B2C and prompt == none THEN throw ex ?